### PR TITLE
aws rds certification expiration check

### PR DESCRIPTION
**Pull request by ianroy-opslevel**

### CODE REVIEW

File: custom_event_checks/opslevel/service_infrastructure_check/aws-rds-certificate-expiration-check.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+      Service **{{ ctx.alias }}** has failed. Please see the list of infrastructure resources below and update databases that still have `rds-ca-2019`. The correct certificate is `rds-ca-rsa2048-g1`
+
+    {% for service in data %}
+    {% if service.aliases[0] == ctx.alias %}
+    {% for infra in service.relatedResources.nodes %}
+    {% if infra != empty %}
+    {% if infra.type == "Database" and infra.rawData["ca_certificate_identifier"] == "rds-ca-2019" %}
+    Infrastructure Resource: **{{ infra.name }}**
+    * ca_certifate_identifier: {{ infra.rawData["ca_certificate_identifier"] }}
+    * http://app.opslevel.com/{{ infra.href }}
```
Comment (rocktavious) [inline custom_event_checks/opslevel/service_infrastructure_check/aws-rds-certificate-expiration-check.yml:42]:
```suggestion
    * http://app.opslevel.com{{ infra.href }}
```

I know it works but i think the extra slash here is unnecessary - you can even see how it generates in the image you linked.

**Review by rocktavious — DISMISSED** [review]
LGTM

File: custom_event_checks/opslevel/service_infrastructure_check/aws-rds-certificate-expiration-check.yml (hunk 1 above)
Comment (ianroy-opslevel) [reply custom_event_checks/opslevel/service_infrastructure_check/aws-rds-certificate-expiration-check.yml]: Great catch, made the change!

**Review by rocktavious — APPROVED** [review]
Change looks good

**Pull request merged**